recipes/nbpresent/post-link.bat

```diff
@@ -0,0 +1 @@
+"%PREFIX%\Scripts\jupyter-nbextension.exe" enable nbpresent --py --sys-prefix && "%PREFIX%\Scripts\jupyter-serverextension.exe" enable --py nbpresent --sys-prefix && if errorlevel 1 exit 1

```

Comment:
I don't think there is a `set -e` equivalent for batch. Believe @msarahan and I had this conversation before even.